core/new-gui/src/app/workspace/service/joint-ui/joint-ui.service.ts

```diff
@@ -4,6 +4,7 @@ import { OperatorSchema } from '../../types/operator-schema.interface';
 
 import * as joint from 'jointjs';
 import { Point, OperatorPredicate, OperatorLink } from '../../types/workflow-common.interface';
+import { optimizeGroupPlayer } from '@angular/animations/browser/src/render/shared';
```

Comment:
remove this if not used.

Reply:
I found that it is necessary to change the color. Also I check the test case you described and it can work on my laptop. maybe we can talk about it later.